spec/services/signups/get_reassessment_spec.rb

```diff
@@ -5,7 +5,12 @@
     subject(:get) { described_class.call(user:) }
     let(:user) { create(:user) }
 
-    let(:signup_assessment) { OpenStruct.new(id: user.incognia_signup_id) }
+    let(:signup_assessment) do
+      OpenStruct.new(
+        id: user.incognia_signup_id,
+        request_id: SecureRandom.uuid
+      )
```

Comment:
We could create a factory with OpenStruct as class, wdyt?
![image](https://user-images.githubusercontent.com/43381819/196725411-233c0f90-c48f-4fc1-937b-fec6e2ae9738.png)

Reply:
Yes! Thanks for forcing me not being lazy. :joy: 

60c2fcd881f3b271818af7b76f57691db0fbb0ae

Reply:
Didn't see it that way 🤣 
:great-success: